docs/windows.md

```diff
@@ -125,127 +158,32 @@ document.
-kind: ConfigMap
-metadata:
-  labels:
-    app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-```
-
-Set the `hostNetwork` option as true in spec of kube-proxy-windows daemonset.
-
-```yaml
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  labels:
-    k8s-app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-spec:
-  selector:
-    matchLabels:
-      k8s-app: kube-proxy-windows
-  template:
-    metadata:
-      labels:
-        k8s-app: kube-proxy-windows
-    spec:
-      hostNetwork: true
-```
+#### Add Windows antrea-agent DaemonSet
 
-For containerd runtime, Set `hostNetwork` as true and add options in spec and command.
+Starting from Antrea 1.13, you need to manually set the `kubeAPIServerOverride` field in the YAML configuration file as the Antrea Proxy `proxyAll` mode is enabled by default.
 
 ```yaml
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  labels:
-    k8s-app: kube-proxy
-  name: kube-proxy-windows
-  namespace: kube-system
-spec:
-  selector:
-    matchLabels:
-      k8s-app: kube-proxy-windows
-  template:
-    metadata:
-      labels:
-        k8s-app: kube-proxy-windows
-    spec:
-      securityContext:
-        windowsOptions:
-          hostProcess: true
-          runAsUserName: "NT AUTHORITY\\SYSTEM"
-      hostNetwork: true
-      serviceAccountName: kube-proxy
-      containers:
-      - command:
-        - pwsh
-        args:
-        - -file
-        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/kube-proxy-windows/run-script.ps1
+    # Provide the address of Kubernetes apiserver, to override any value provided in kubeconfig or InClusterConfig.
+    # Defaults to "". It must be a host string, a host:port pair, or a URL to the base of the apiserver.
+    kubeAPIServerOverride: "10.10.1.1:6443"
+
+    # Option antreaProxy contains AntreaProxy related configuration options.
+    antreaProxy:
+      # ProxyAll tells antrea-agent to proxy ClusterIP Service traffic, regardless of where they come from.
+      # Therefore, running kube-proxy is no longer required. This requires the AntreaProxy feature to be enabled.
+      # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
+      # apiserver directly.
+      proxyAll: true
 ```
 
-Then apply the `kube-proxy.yml`.
-
 ```bash
-kubectl apply -f kube-proxy.yml
+# Example:
+KUBE_APISERVER=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}') && \
+curl -sL https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-windows-containerd.yml | \
+sed "s|.*kubeAPIServerOverride: \"\"|    kubeAPIServerOverride: \"${KUBE_APISERVER}\"|g" | \
+kubectl apply -f -
 ```
 
-#### Add Windows antrea-agent DaemonSet
-
 Now you can deploy antrea-agent Windows DaemonSet with Docker runtime by applying file `antrea-windows.yml`.
```

Comment:
Do not understand the following contents. The previous command already deploys antrea-agent right? Why the following are still talking about antrea-agent deployment. Could you check?

Reply:
Updated, since 1.13 I think we should remove commands without updating kubeAPIServerOverride because the default proxyall value is true.